de-de/html-de.html.markdown

```diff
@@ -111,7 +111,7 @@ Dieser Artikel ist bedacht darauf, nur HTML Syntax und nützliche Tipps zu geben
 
 ## Verwendung
 
-HTML Dateien enden mit `.html`.
+HTML Dateien enden mit `.html` oder mit `.htm`. Der Mime Typ ist meist `text/html`.
```

Comment:
`htm`? Benutzt jemand das noch immer? Und ist es in HTML 5 erlaubt?

Reply:
eigentlich geht sogar jegliche Dateiendung. Der Mime Typ ist hier ausschlaggebender. Und ja es wird benutzt.